### DISCUSSION TITLE
DOSBox WASM core and more...

### DISCUSSION THREAD

**Discussion by superpippoub77** [EmulatorJS/EmulatorJS#518] [General]
Hi,
this is a big project, i have some questions:

1. there is a list of games for _mame2003_
2. there is a list of _arcade_ games
3. Is there a **DOSBox-wasm.data**? (I liked playing the old games), **if it exists where can I download it?**

Thank you all.

**Comment by ethanaobrien** [comment]
>there is a list of games for mame2003

Yes, it uses the mame 2003 libretro core so [this](https://buildbot.libretro.com/compatibility_lists/cores/mame2003/mame2003.html) should be the list.

>there is a list of arcade games

there are 2 arcade cores. One being `fbalpha2012_cps1` and one is `fbalpha2012_cps2`
[List for `fbalpha2012_cps1` core](https://github.com/libretro/fbalpha2012_cps1/blob/master/gamelist.txt)
[List for `fbalpha2012_cps2` core](https://github.com/libretro/fbalpha2012_cps2/blob/master/gamelist.txt)

>Is there a DOSBox-wasm.data

Not at the moment, due to some incompatibilities. Sorry

**Reply by superpippoub77** [reply]
Hi @ethanaobrien,

another question...
if i want to load a game in div and change the content in the same div is it possible?.**..because 99% emulator loading core is stuck**.

**First ok:**

![image](https://github.com/EmulatorJS/EmulatorJS/assets/26119108/f101f965-9bba-4d8e-b38b-948c04942aae)

**After KO:**

![image](https://github.com/EmulatorJS/EmulatorJS/assets/26119108/13c7d951-acc6-4e70-8bff-98fa224eca40)

The page is locked.

:-(

Thx;
